src/emptyState/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import cc from 'classcat'
+import Title from 'title'
+import style from './style'
+
+interface EmptyState {
+  readonly className?: Classcat.Class
+  readonly image: string
+  readonly text: string
+  readonly button?: JSX.Element
+}
+
+const EmptyState = ({ className, image, text, button}: EmptyState) => (
+  <div className={cc(['kirk-empty-state', className])}>
+    <img src={image} alt="" />
+    <Title>{text}</Title>
+    {button}
```

Comment:
`button` prop, but this is not enforced I could pass a Modal in there and it will still render.